components.yaml

```diff
@@ -31,7 +31,7 @@ GMAO_Shared:
   remote: ../GMAO_Shared.git
   tag: v1.9.6
   sparse: ./config/GMAO_Shared.sparse
-  develop: main
+  develop: feature/adarmenov/#122-wmma
```

Comment:
@adarmenov I think you meant to put this in place of `tag:` not `develop:`. So that when you `mepo clone` you get your branch. As such you'd want:
```
branch: feature/adarmenov/...
```
in the places you want your branches instead of the `tag:`.

`develop:` wouldn't change.

Reply:
I could also push this change up if you prefer.

Reply:
@mathomp4 - should be addressed with commit https://github.com/GEOS-ESM/GEOSgcm/commit/530d776a91edfc3b7135add1660905bc71196d7e

Reply:
@mathomp4 - hopefully commit https://github.com/GEOS-ESM/GEOSgcm/commit/0f9c13403b4d1a0107f8eb772d826346b33a0738 is acceptable

Reply:
That's got it!